scripts/extract_helper_names.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+#
+# This file is distributed under the MIT License. See LICENSE.md for details.
+#
+
+set -euo pipefail
+
+printf "Name\\n"
+
+(
+  for FILE in "$@"; do
+    (
+      if [[ "${FILE}" =~ ^.+\.bc$ ]]; then
+        cat ${FILE}
+
+      elif [[ "${FILE}" =~ ^.+\.ll$ ]]; then
+        llvm-as "${FILE}" -o -
+
+      else
+        echo "\"${FILE}\" does not have a supported llvm-module extension. Is it even a module?" 1>&2
+        exit 1
+      fi
+    ) |
+
+    # TODO: remove `--defined-only` to pick up all the symbols once they are all
+    # prefixed correctly.
```

Comment:
IIRC this is about not banning `free`, correct?

If so, stress 1) why we do this (i.e., avoid banning `free`), 2) what are the consequences of not including declared symbols (i.e., `void free(void *)` and `void free(char *)` will lead to compilation errors) and 3) that this is a temporary hack and that the solution we foresee is to include also the declared functions, but prefix them all, reducing the risk of clashing.

Reply:
Done:

```
// Note that namespace collisions can only occur on invalid models (there are
// exactly two use cases of it - verification and post-importer fixes), as such
// there's no need to expose this logic outside of `libModel` - as it's expected
// that all the dependencies work exclusively on valid models.
//
// If you find yourself with a model containing duplicated identifiers after
// an analysis, instead of exposing this, take a look at `fix-model` pass (or
// one of its components) instead.
```